src/libraries/Common/src/System/HResults.cs

```diff
@@ -26,6 +26,7 @@ internal static partial class HResults
     {
         internal const int S_OK = unchecked((int)0x00000000);
         internal const int S_FALSE = unchecked((int)0x1);
+        internal const int E_ACCESSDENIED = unchecked((int)0x80070005);
```

Comment:
Nit: Adding it front of `E_BOUNDS` would follow the ordering that seems to be maintained in the file so far.